### PR TITLE
Update locations of static data on Cheyenne, Hera, and Jet

### PR DESCRIPTION
**Pull request by mkavulich**
## DESCRIPTION OF CHANGES: 
Several paths in the machine-specific files point to locations in user paths or old locations of static data. This PR updates paths of static data in regional_workflow/ush/machine/ to point to the official, centralized locations on Cheyenne, Hera, and Jet.

## TESTS CONDUCTED: 
Ran the following suite of end-to-end tests on Cheyenne and Jet prior to the latest ufs-weather-model hash update. All passed. This list of tests was chosen because all of these tests are known to succeed on all tested platforms, and this tests a variety of input and boundary condition types.

- grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
- grid_RRFS_CONUS_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
- grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
- grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
- grid_RRFS_CONUS_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
- grid_RRFS_CONUS_25km_ics_HRRR_lbcs_HRRR_suite_RRFS_v1beta
- grid_RRFS_CONUS_25km_ics_HRRR_lbcs_RAP_suite_HRRR
- grid_RRFS_CONUS_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
- grid_RRFS_CONUS_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta

On Hera, I ran tests with the latest SRW hash, which included the updated weather model. Because of this, many tests could not be generated due to using old, removed CCPP suites (see issue #668). To get around this issue, I tested with the fixes from #697 incorporated into my branch. With those extra commits, all "get_extrn_ics" and "get_extrn_lbcs" tasks completed successfully, which indicates that all data is in its correct place.

## DEPENDENCIES:
This will require a ufs-srweather-app PR to update the regional_workflow hash, will update when that PR is opened.

## ISSUE (optional): 
Will resolve a few issues in #673, many remain however.

### CODE REVIEW

File: ush/machine/cheyenne.sh [hunk 1]
```diff
@@ -68,6 +68,6 @@ NDAS_OBS_DIR=${NDAS_OBS_DIR:-"/glade/p/ral/jntp/UFS_SRW_app/develop/obs_data/nda
 MET_BIN_EXEC=${MET_BIN_EXEC:-"bin"}
 
 # Test Data Locations
-TEST_PREGEN_BASEDIR="/glade/p/ral/jntp/UFS_CAM/FV3LAM_pregen"
-TEST_COMINgfs="/glade/scratch/ketefian/NCO_dirs/COMGFS"
+TEST_PREGEN_BASEDIR="/glade/p/ral/jntp/UFS_SRW_app/FV3LAM_pregen"
+TEST_COMINgfs="/glade/p/ral/jntp/UFS_SRW_app/COMGFS"
```
Comment (christinaholtNOAA) [inline ush/machine/cheyenne.sh:72]: I was just taking a look at the COMGFS directories and noticed that they have nemsio files from GFS. Is that still a relevant test for testing COMGFS input mechanisms? Aren't we initializing near real time runs from either GFS netcdf or grib2 files?

Comment (mkavulich) [reply ush/machine/cheyenne.sh:72]: I'll defer to others to answer this question, I'm not familiar with how "nco mode" should be tested. @gsketefian or @JeffBeck-NOAA maybe know better?

Comment (gsketefian) [reply ush/machine/cheyenne.sh:72]: We can (and should) add netcdf and grib2 files to the set of nco-mode tests.  We have no guidance on how long we should keep supporting old/discontinued formats like nemsio.  I have left it up to the EMC developers to maintain the nco-mode tests.  @chan-hoo @BenjaminBlake-NOAA @RatkoVasic-NOAA any thoughts?

**Review by JeffBeck-NOAA — APPROVED** [review]

File: ush/machine/cheyenne.sh (hunk 1 above)
Comment (chan-hoo) [reply ush/machine/cheyenne.sh:72]: @christinaholtNOAA, @gsketefian, Sorry for my late response. I'll modify the we2e scripts that use 'nemsio' files for the 'nco' mode soon. Once it is done, we can remove the nemsio files from the COMGFS directory.

Comment (mkavulich) [reply ush/machine/cheyenne.sh:72]: @chan-hoo That's good news! If you ping me once that PR is merged I can remove those directories from the platforms I have access to (Cheyenne, Hera, Jet, and Orion).

Comment (chan-hoo) [reply ush/machine/cheyenne.sh:72]: @mkavulich, Sure. I'll let you know.
